Clinical-Genomics/scout
# Fix #1597

This PR adds a functionality or fixes a bug.

**How to test**:
1. load research for a full case on stage
1. ensure both SV and SNV research can be opened.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
